cpp/daal/src/algorithms/dtrees/forest/classification/oneapi/df_classification_train_hist_kernel_oneapi.h

```diff
@@ -170,8 +171,13 @@ class ClassificationTrainBatchKernelOneAPI<algorithmFPType, hist> : public daal:
     const size_t _maxBins                 = 256;
     const size_t _reduceLocalSizePartHist = 64;
 
+    const size_t _minPreferableLocalSizeForPartHistKernel = 32;
+
     const size_t _maxPartHistCumulativeSize      = 805306368; // 768 Mb
-    const size_t _minRowsBlocksForMaxPartHistNum = 1024;
+    const size_t _maxTreeObservationsMapSize     = 134217728; // 128 Mb
```

Comment:
Do these values depend on the GPU architecture?

Reply:
yes, going to add this dep with next MRs

Reply:
added dependency on available mem